docs/getting-started-kotlin/build.gradle.kts

```diff
@@ -25,6 +25,11 @@ tasks.withType<KotlinCompile> {
     kotlinOptions.jvmTarget = "1.8"
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
It's not directly related, but should the following be fixed?
https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb-by-using-kotlin.md#install-a-jdk

Reply:
[This document was moved to the doc site](https://github.com/scalar-labs/docs-internal-scalardb/blob/main/docs/en-us/getting-started-with-scalardb-by-using-kotlin.mdx) and shows correct JDK requirements. My understanding is that we should not update the document on the ScalarDB repo anymore. 
What do you think ?